staging/src/k8s.io/apiserver/plugin/pkg/authenticator/token/webhook/webhook_test.go

```diff
@@ -398,168 +397,3 @@ func TestWebhookTokenAuthenticator(t *testing.T) {
 		}
 	}
 }
-
-type authenticationUserInfo v1beta1.UserInfo
-
-func (a *authenticationUserInfo) GetName() string     { return a.Username }
-func (a *authenticationUserInfo) GetUID() string      { return a.UID }
-func (a *authenticationUserInfo) GetGroups() []string { return a.Groups }
-
-func (a *authenticationUserInfo) GetExtra() map[string][]string {
-	if a.Extra == nil {
-		return nil
-	}
-	ret := map[string][]string{}
-	for k, v := range a.Extra {
-		ret[k] = []string(v)
-	}
-
-	return ret
-}
-
-// Ensure v1beta1.UserInfo contains the fields necessary to implement the
-// user.Info interface.
-var _ user.Info = (*authenticationUserInfo)(nil)
-
-// TestWebhookCache verifies that error responses from the server are not
-// cached, but successful responses are. It also ensures that the webhook
-// call is retried on 429 and 500+ errors
-func TestWebhookCacheAndRetry(t *testing.T) {
```

Comment:
Maybe keep this test and update it with token cache. Just to ensure behavior doesn't change.